@navikt/core/css/table.css

```diff
@@ -141,3 +141,76 @@
   font-size: 1rem;
   flex-shrink: 0;
 }
+
+.navds-table__expandable-row--open .navds-table__header-cell,
+.navds-table__expandable-row--open .navds-table__data-cell {
+  border-color: var(--navds-semantic-color-border-muted);
+}
+
+.navds-table__expandable-row--open .navds-table__toggle-expand-cell--open {
+  border-color: transparent;
+}
+
+.navds-table__toggle-expand-cell {
+  padding: 0;
+  width: 56px;
+}
+
+.navds-table--small .navds-table__toggle-expand-cell {
+  padding: 0;
+  width: 36px;
```

Comment:
Bare en tanke, men bør tabell-høyder bli trekt ut i en token? Kan da åpne opp for lettere custom tabeller der størrelsen blir justert av tokens.

Reply:
Ikke en dum tanke! Enn så lenge begrenser vi størrelser på table til `"medium"` og `"small"`, men om det blir flere er tokens sikkert lurt.